app/services/manifest_builder.rb

```diff
@@ -47,7 +47,7 @@ def initialize(resource)
     # Stringify the Resource by delegating to the header within the Decorator
     # @return [String]
     def to_s
-      decorate.header
+      resource.title.map(&:to_s)
```

Comment:
looks like the yard docs are now out of sync with the method here.

Reply:
Yup! Fixed.